### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by yoannpicquenot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+# Routes
+
+Routes                       | Description
+---------------------------- | -----------
+`GET /api/:version/gpus`     | Return an array containing all existing gpus
+`GET /api/:version/gpus/:id` | Return a specific gpu with a specific id
+`PUT /api/:version/gpus`     | Add a new gpu
+
+#### Details for a Gpu can be found [here](./v1/models/gpu/GPU.md)
```
Comment (Quentin01) [inline README.md]: Why are you using `####`? It's for title, not to highlight something

Comment (Quentin01) [reply README.md]: And do you really want to just copy/paste the schema of the model? The user still doesn't know what is the exact JSON for each routes.

**Review by Quentin01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yoannpicquenot) [reply README.md]: There's only one JSON model for each route.

Comment (yoannpicquenot) [reply README.md]: I don't have a better solution

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+# Routes
+
+Routes                       | Description
+---------------------------- | -----------
+`GET /api/:version/gpus`     | Return an array containing all existing gpus
+`GET /api/:version/gpus/:id` | Return a specific gpu with a specific id
+`PUT /api/:version/gpus`     | Add a new gpu
+
+Details for a Gpu can be found [here](./v1/models/gpu/GPU.md)
```
Comment (Quentin01) [inline README.md]: Because the markdown file is just a copy/paste from the Mongoose schema, maybe you can just add a link to the JS file instead?

**Review by Quentin01 — CHANGES_REQUESTED** [review]

**Comment by yoannpicquenot** [comment]
@Quentin01 is it mergeable?

**Review by Quentin01 — APPROVED** [review]

**Pull request merged**